SeaPublicWebsite.BusinessLogic/ExternalServices/Bre/RecommendationService.cs

```diff
@@ -540,5 +543,16 @@ private static bool GetBreOutsideSpace(HasOutdoorSpace? hasOutdoorSpace)
                 _ => throw new ArgumentOutOfRangeException()
             };
         }
+        
+        private static bool? GetBrePvPanels(SolarElectricPanels? solarElectricPanels)
+        {
+            return solarElectricPanels switch
+            {
+                SolarElectricPanels.Yes => true,
+                SolarElectricPanels.No => false,
+                SolarElectricPanels.DoNotKnow => null,
```

Comment:
Null and False lead to the same outcome with the API. I've left it this way because I think when we get the hint data, we want to override the user's option on a "don't know" with the open EPC information. 

At least that was my understanding of the documents on the ticket.

Reply:
Ah! No that's fair, I'll change it over. I was looking at the attached document's open epc thing

Reply:
this change has been made. left a fallback 'null' in the case of no solar pv answer